modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkIndexByScrollResponse.java

```diff
@@ -173,7 +197,8 @@ public String toString() {
         StringBuilder builder = new StringBuilder();
         builder.append("BulkIndexByScrollResponse[");
         builder.append("took=").append(took).append(',');
-        status.innerToString(builder, false, false);
+        builder.append("timed_out=").append(timedOut).append(',');
+        status.innerToString(builder, new MapParams(includeParams));
```

Comment:
I think I'd just let deleted and updated come out in the toString rather than worry about that. I think it was a mistake that I did it. I don't think it is worth the complexity of the map. It is just a couple of fields that we can ignore.